src/SFA.DAS.AANHub.Api/Common/ActionResponseControllerBase.cs

```diff
@@ -0,0 +1,44 @@
+﻿using FluentValidation.Results;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.AANHub.Application.Mediatr.Common;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+
+namespace SFA.DAS.AANHub.Api.Common
+{
+    public class ActionResponseControllerBase : ControllerBase
```

Comment:
Didn't see coverage specifically on this?

Reply:
No, as the methods are protected (which I believe is the best level of access for base class methods), you can't test them directly.

In a situation like this, I always find it's better to test implementation, rather than the code directly. Plus, we'd need to choose a response type (such as CreateApprenticeCommandResponse) to use anyway, so we'd be duplicating the tests in CreateApprenticeCommandTests if we raised the access level anyway.